src/addrman.h

```diff
@@ -569,12 +569,13 @@ class CAddrMan
     //! Mark an entry as currently-connected-to.
     void Connected(const CService &addr, int64_t nTime = GetTime())
     {
-        LOCK(cs);
-        Check();
-        Connected_(addr, nTime);
-        Check();
+        {
+            LOCK(cs);
+            Check();
+            Connected_(addr, nTime);
+            Check();
+        }
```

Comment:
Presumably this is to do with dropping the lock?  It seems redundant, but I suppose one might later add code outside of the inner block, but still in the function.

Reply:
By 'this' you mean the double braces? I've seen them around a lot, where there's a block that contains nothing except for another block. As far as I can tell it's semantically meaningless, but I could be missing something, or they're some sort of human-meaningful convention I've never heard of.